### PR TITLE
Partition config example

### PR DESCRIPTION
**Pull request by heitorPB**
**Please provide description of the purpose of this pull request, as well as its
motivation and context.**

- Makefile: add "docstrings" to the tests target
- release.sh: add an empty line for the changes. This is used in some
  places to separate the title from the description.
- slurmd:config: add example to partition-config

**How was the code tested?**

...

Final checklist
- [x] I am the author of these changes, or I have the rights to submit them.
- [ ] I added the relevant changed to the README and/or documentation.
- [x] I self reviewed my own code.
- [ ] I updated the `CHANGELOG` according to the [contributing
  guide](https://omnivector-solutions.github.io/osd-documentation/master/contributing.html#changelog)

### CODE REVIEW

File: charm-slurmd/config.yaml [hunk 1]
```diff
@@ -27,8 +27,11 @@ options:
     type: string
     default: ""
     description: >
-      Extra partition configuration specified as a space separated `key=value`
-      single line.
+      Extra partition configuration, specified as a space separated `key=value`
+      in a single line.
+
+      Example usage:
+      $ juju config slurmd partition-config="DefaultTime=45:00 MaxTime=1:00:00"
```
Comment (jamesbeedy) [inline charm-slurmd/config.yaml:34]: Do you think it is worth linking to https://slurm.schedmd.com/slurm.conf.html#SECTION_PARTITION-CONFIGURATION

Comment (jamesbeedy) [reply charm-slurmd/config.yaml:34]: It may clutter the description though….. Just a thought.

**Review by jamesbeedy — APPROVED** [review]
+1

File: charm-slurmd/config.yaml (hunk 1 above)
Comment (heitorPB) [reply charm-slurmd/config.yaml:34]: We could link that in the docs: https://omnivector-solutions.github.io/osd-documentation/master/operations/partitions.html

Comment (heitorPB) [reply charm-slurmd/config.yaml:34]: Done: https://github.com/omnivector-solutions/osd-documentation/pull/44

**Pull request merged**